usage.md

```diff
@@ -0,0 +1,46 @@
+
+<h1><img align="center" height="50" src="logo.png"> nbgitpuller extension</h3>
+
+
+> 🚨Setup and tutorial instructions for using the nbgitpuller chrome extension for assignment distribution
+
+## ❗️setup - nbgitpuller extension
+
+Upon installation, select the puzzle piece located in the upper-right corner of your Chrome tab.
+
+![](usage-media/setup1.png)
+
+Click on the pin button to make sure nbgitpuller stays pinned at the top of the screen.
+
+![](usage-media/setup2.png)
+
+From now on, you can access nbgitpuller directly from the Chrome toolbar.
+
+![](usage-media/setup3.png)
+
+## ❗️ tutorial - nbgitpuller extension
+
+To begin, navigate to the relevant folder within the assignment’s GitHub repository containing notebooks. Select the nbgitpuller extension in the browser, and you will have options to enter the Hub URL and the interface to launch the notebook.
+
+Enter the URL to the hub that you use for administering your assignments. E.g.: "[https://datahub.berkeley.edu](https://www.google.com/url?q=https://datahub.berkeley.edu&sa=D&source=editors&ust=1704918064934417&usg=AOvVaw1lIlYFhcEVYh8wOMxgpO84)."
+
+![](usage-media/tut1.png)
+![](usage-media/tut2.png)
+
+> 🚨 Note: Once the JupyterHub URL is specified once, nbgitpuller will autofill it the next time, no need to keep inputting it.
+
+Following that, indicate the notebook interface that students should use for working with the notebooks. The options are Classic Notebook, Retrolab, JupyterLab, and RStudio for R assignments. You can launch Python notebooks using Jupyter interfaces such as Classic Notebook, RetroLab, and JupyterLab.
```

Comment:
@ranashreyas Do you want to fix usage.md updates as a separate PR or in the same PR?